src/tools/stereoCalib/src/stereoCalibThread.cpp

```diff
@@ -33,7 +33,8 @@ stereoCalibThread::stereoCalibThread(ResourceFinder &rf, Port* commPort, const c
     int tmp=stereoCalibOpts.check("MonoCalib", Value(0)).asInt();
     this->stereo= tmp?false:true;
     this->camCalibFile=rf.getHomeContextPath().c_str();
-
+    this->standalone = rf.check("standalone") &&
```

Comment:
What's that? The check is replicated twice: one should be sufficient 😉

Reply:
That's just to use the parameter as a flag and write in command line _--standalone_ rather than _--standalone true/false_

Reply:
Interesting; did you check it's behaving as expected?